src/components/icon/styles.css

```diff
@@ -12,36 +27,73 @@
 	.icon.size_96x96::before {
 		display: block;
 		content: '';
-		width: 96px;
-		height: 96px;
 		top: 0;
 		left: 0;
+		bottom: 0;
+		right: 0;
 		position: absolute;
 		background-color: #f4f9fb;
 		border-radius: 48px;
 		z-index: -1;
 		}
 
+		.icon.size_96x96:hover:before {
+			top: -1px;
+			bottom: -1px;
+			left: -1px;
+			right: -1px;
+			border: #b9d5e8 solid 1px;
+			border-radius: 48px;
+		}
 .icon.size_120x120 {
 	width: 120px;
 	height: 120px;
 	background-repeat: no-repeat;
 	background-position: 50% 50%;
 	background-size: 64px 64px;
 	}
-	.icon.size_120x120::before {
-		display: block;
-		content: '';
-		width: 120px;
-		height: 120px;
-		top: 0;
-		left: 0;
-		position: absolute;
-		background-color: #f4f9fb;
-		border-radius: 60px;
-		z-index: -1;
+		.icon.size_120x120::before {
+			display: block;
+			content: '';
+			top: 0;
+			left: 0;
+			bottom: 0;
+			right: 0;
+			position: absolute;
+			background-color: #f4f9fb;
+			border-radius: 148px;
```

Comment:
From where 148px border-radius is coming from?

Reply:
adjusted to 60px = 1/2 * 120 (half the icon size)

Reply:
I don't see any changes, where is it adjusted?